Libs/RaptureCore/src/test/java/rapture/kernel/SysApiTest.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2011-2016 Incapture Technologies LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package rapture.kernel;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.io.File;
+import java.util.Map;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import rapture.common.ConnectionInfo;
+import rapture.common.RaptureConstants;
+import rapture.common.RaptureURI;
+import rapture.common.Scheme;
+import rapture.common.api.SysApi;
+import rapture.common.connection.ConnectionType;
+
+public class SysApiTest extends AbstractFileTest {
+
+    private static final String auth = "test" + System.currentTimeMillis();
+    private static final String REPO_USING_FILE = "REP {} USING FILE {prefix=\"/tmp/" + auth + "\"}";
+    private static final File temp = new File("/tmp/" + auth);
+    private static final String scriptAuthorityURI = "script://" + auth;
+    private static final String scriptURI = scriptAuthorityURI + "/For/A/Jesters/Tear";
+
+    private static ScriptApiImpl scriptImpl;
+
+    String saveRaptureRepo;
+    String saveInitSysConfig;
+
+    @BeforeClass
+    static public void setUp() {
+        AbstractFileTest.setUp();
+        config.RaptureRepo = REPO_USING_FILE;
+        config.InitSysConfig = "NREP {} USING FILE { prefix=\"/tmp/" + auth + "/sys.config\"}";
```

Comment:
unit tests should not write to the filesystem and leave new files there on every run.  use in-memory instead